packages/core/test/financial-templates/common/financial-product-libraries/KpiOptionsFinancialProductLibrary.js

```diff
@@ -31,7 +31,7 @@ contract("KpiOptionsFinancialProductLibrary", function() {
     );
   });
   describe("price transformation", () => {
```

Comment:
I know this test script is copied from another FPLib contract but thoughts on adding a unit test to check if a revert occurs when the transformedPrice is NOT set yet?

Reply:
Would this work in the non-1e18 case when the price is passed as a `FixedPoint.Unsigned`?

Reply:
@nicholaspai hmm, it actually looks like it should be reverting but isn't, since `setFinancialProductTransformedPrice` is never called. Looking into it.